rfc-086-draft-stack-rummager.md

```diff
@@ -0,0 +1,37 @@
+# Draft Stack Rummager
+
+## Summary
+
+RFC 86 proposes setting up a instance of [Rummager][rummager] for the
+[draft stack][draft-stack-docs].
+
+## Problem
+
+The [draft stack (content preview)][draft-stack-docs] part of GOV.UK
+relies in places on the [rummager service (Search API)][rummager].
+
+There is only one instance of this service, which is primarily used
+for the live stack, but is also currently used from some services on
+the draft stack (see [govuk-puppet][puppet-frontend-rummager-config]).
+
+Rummager may return some draft content, depending on the internal
```

Comment:
This is not draft content itself, it's links to draft content (eg the mainstream_browse_pages, topics, organisations fields of other documents). This could still have unintended consequences for the live stack though.

Reply:
I've attempted to clarify this a bit in 6a9f6433e4f05e42a235695acb282f6ef03f7845